ArduCopter/mode_guided.cpp

```diff
@@ -535,7 +535,8 @@ void ModeGuided::pos_control_run()
 
     // if not armed set throttle to zero and exit immediately
     if (is_disarmed_or_landed()) {
-        make_safe_spool_down();
+        // do not spool down tradheli when on the ground with motor interlock enabled
+        make_safe_ground_handling(copter.is_tradheli() && motors->get_interlock());
```

Comment:
In fact, you could even leave out the, "motors->get_interlock()" part because I think at the motors layer it will force it to shutdown when interlock is enabled.  Still, it does no harm and I think it's fine as-is.

Reply:
Yes I thought about that but I wanted to leave the control with the motors class rather than have this try to override. Anyway if you are happy with it like this then I will leave it that way.

Reply:
@bnsgeyer, yes, find with me thanks.